src/Equinox.MessageDb/MessageDb.fs

```diff
@@ -1,5 +1,6 @@
 ﻿namespace Equinox.MessageDb
 
+open System.Diagnostics
```

Comment:
topological sorting and/or special casing of System pains me so much!

Reply:
haha, I'm just in camp "let Rider put it where it wants" I'll sort it